Sources/Apodini/Configurations/HTTP2Configuration.swift

```diff
@@ -27,20 +28,37 @@ public class HTTP2Configuration: Configuration {
     private var certData: Data?
     private var keyData: Data?
 
-    public init() {
+    public convenience init() {
+        self.init(arguments: CommandLine.arguments)
+    }
+
+    init(arguments: [String]) {
         do {
-            for arg in CommandLine.arguments {
-                if arg.starts(with: "--cert=") {
-                    certData = try Data(contentsOf: URL(fileURLWithPath: arg.substring(from: 7)))
-                } else if arg.starts(with: "--key=") {
-                    keyData = try Data(contentsOf: URL(fileURLWithPath: arg.substring(from: 6)))
-                }
-            }
+            var commandInput = CommandInput(arguments: arguments)
+            try detect(from: &commandInput)
         } catch {
             print("Cannot read certificate / key file provided via command line. Error: \(error)")
         }
     }
 
+    func detect(from commandInput: inout CommandInput) throws {
+        struct Signature: CommandSignature {
+            @Option(name: "cert", short: "c", help: "Path of the certificate")
+            var certPath: String?
+            @Option(name: "key", short: "k", help: "Path of the key")
+            var keyPath: String?
+        }
+
+        let signature = try Signature(from: &commandInput)
+
+        if let certPath = signature.certPath, let keyPath = signature.keyPath {
+            certData = try Data(contentsOf: URL(fileURLWithPath: certPath))
+            keyData = try Data(contentsOf: URL(fileURLWithPath: keyPath))
+        } else {
+            print("Cannot read certificate / key file provided via command line")
```

Comment:
I found another little one: Please use the logger instead of print (might be more cases than just this one).

Reply:
At this point the logger / Apodini app does not yet exist. But I found some other instances of print and fixed those.

Reply:
I made the errors fatalErrors.